### DISCUSSION TITLE
Basic protobuf support?

### DISCUSSION THREAD

**Discussion by dfederighi** [suren-atoyan/monaco-react#334] [General]
Hi ... as I understand, the underlying Monaco-editor supports basic syntax highlighting for protobuf and I wondered when the react version would include that?

**Comment by suren-atoyan** [comment]
`monaco-editor` doesn't support syntax highlighting for `protobuf` (`monaco-react` too), but you can implement it 🙂 

please check [this](https://codesandbox.io/s/protobuf-nmfwph?file=/src/App.js)
